src/types/types.d.ts

```diff
@@ -203,15 +201,15 @@ export interface ClaimableAsset extends Asset {
   claimants: any[];
 }
 
-export interface ActiveAsset {
+export interface ActiveAssetAction {
   id: string;
```

Comment:
Do you think `id` makes sense here, I think `assetCode` might be better, unless you think we could have `id`s that aren't asset codes in the future.

Reply:
Yeah, I thought about that and went with a more flexible `id`, which happens to be `assetCode` but it might be something else? 🤔 I'm OK either way. @piyalbasu, how do you feel about it?

Reply:
Since it doesn't seem like there are instances of `id` not being an `assetCode`, I like simply calling it `assetCode` now as it's a more descriptive property name.

Reply:
👍 I'll update, thank you!